library/src/ui/index.js

```diff
@@ -1 +1,72 @@
-window.CoNDeT.ui = {}
\ No newline at end of file
+window.CoNDeT.ui = {
+    BaseComponent: {
+        state: {
+
+        },
+        common: {
+
+        },
+        init: function(common, props) {
+            this.ref = this.onInit();
+        },
```

Comment:
Do `onInit` należy podać `common` i `props`. Dodatkowo `common` należy zapisać w obiekcie nadpisując wartość `this.common`